commander.yaml

```diff
@@ -276,7 +288,7 @@ tests:
         - IDENTIFIER            http://integration-test-api-def2
         - SCOPES                read:todos write:todos
         - TOKEN LIFETIME        86400
-        - ALLOW OFFLINE ACCESS  false
+        - ALLOW OFFLINE ACCESS  ✗
```

Comment:
cool. are we ok with the --format=JSON output? true || false ?

Reply:
Yes, the JSON output retains the `true` and `false`.